server/graphql/v2/query/index.js

```diff
@@ -32,6 +33,7 @@ const query = {
   account: AccountQuery,
   accounts: AccountsCollectionQuery,
   activities: ActivitiesCollectionQuery,
+  activityTimeline: ActivityTimelineQuery,
```

Comment:
What is the difference between `activities` and `activityTimeline` ? Could we reuse `activities`?

Reply:
Or have it served has a resolver under `account` like `account.activities`?

Reply:
I thought it didn't make much sense to have this as a property of the `Account` since this should take into consideration the context of the user resolving it when deciding what should be returned.

The existing `ActivitiesCollectionQuery` feels a bit raw for what we want in the Timeline. It works great for listing the activities that are directly related to a collective (activity log) but the timeline will be a bit more complex.

`activityTimeline` is a sort of opinionated resolver for activities that will return activities from different entities related to the account. Since we will be deciding what to show specific accounts based on different conditions, and this can potentially grow in complexity, I would like to isolate this logic to the backend and keep the arguments to a minimum. For this reason, I added a new resolver dedicated to the timeline.

The name can be a bit confusing as it suggests it is just another activity query, should we rename?